components/AvalancheProblemCard.tsx

```diff
@@ -44,7 +42,7 @@ export const AvalancheProblemCard: React.FunctionComponent<AvalancheProblemCardP
       </View>
       {problem.media.type === MediaType.Image && problem.media.url !== null && <AvalancheProblemImage media={problem.media} />}
       <View style={styles.content}>
-        <RenderHTML source={{html: problem.discussion}} contentWidth={width} />
+        <HTMLRenderer source={{html: problem.discussion}} />
```

Comment:
super-nit: seems like component names should be more like nouns of what they are, e.g. `<RenderedHTML/>` rather than the actor doing the rendering

Reply:
WDYT about just `<HTML>`?

Reply:
SGTM